test/render/fixtures/attrs/expected.html

```diff
@@ -1 +1 @@
-<div data-encoding="&quot;hello&quot;" style="background-color: #FF0000; &lt;test>" class="my-div" checked>Hello World!</div>
```

Comment:
This test should not need to change, doing so would be a breaking change. If the order of the dynamic tags moves to the end this should be fixed.